builder/vagrant/builder.go

```diff
@@ -94,8 +94,6 @@ type Config struct {
 	Provider string `mapstructure:"provider" required:"false"`
 	// Options for the "vagrant init" command
 
-	// What vagrantfile to use
```

Comment:
Nit: we should mention this in the PR name for the git history of removing this field

Reply:
In general we don't squash/merge, this is already in a separate commit, so I don't think this is an issue